lib/iex/lib/iex/helpers.ex

```diff
@@ -1389,4 +1389,13 @@ defmodule IEx.Helpers do
         {:error, :nofile}
     end
   end
+
+  defp get_beam_and_path(module) do
```

Comment:
I wonder if it would be best to make this more DRY and make this a non-documented helper that could be used across the codebase?

Reply:
I checked and each of them return slightly different data. Plus, we typically don't use "private" APIs between applications, so we would need to duplicate between IEx and Elixir anyway, so I think we can skip for now!